node_modules/col-core/lib/email.js

```diff
@@ -52,6 +52,26 @@ var EVENT_NAMES = module.exports.EVENT_NAMES = {
   'EMAIL_SENT': 'emailSent'
 };
 
+// Variable that maps the original image paths to their hashed counter parts (if any)
+var imageManifest = {};
+
+// Cache the image manifest
+var manifestPath = path.resolve(__dirname, '..', '..', '..', config.get('apache.documentRoot'), 'images-rev-manifest.json');
```

Comment:
Isn't it easier to just keep `documentRoot` the way it was as it points to where the static files are? If it's a production build, `images-rev-manifest.json` should be there. It would avoid having to hardcode going up 3 levels.